pallets/creditcoin/src/migrations/mod.rs

```diff
@@ -30,7 +29,6 @@ pub(crate) fn migrate<T: Config>() -> Weight {
 		&v3::Migration::<T>::new(),
 		&v4::Migration::<T>::new(),
 		&v5::Migration::<T>::new(),
-		&v6::Migration::<T>::new(),
```

Comment:
this will mess up the migration index vs storage version parity, see 10 lines below.

I suggest leaving v6 empty with an explicit comment that it has been reverted.